src/builtins.c

```diff
@@ -0,0 +1,154 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   builtins.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: dbejar-s <dbejar-s@student.hive.fi>        +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/08/06 14:24:52 by dbejar-s          #+#    #+#             */
+/*   Updated: 2024/08/08 01:46:29 by dbejar-s         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
+extern int	g_state;
+
+//si gestionamos g_state, esta funcion deberia retornar un int
+
+void    ft_pwd(void) 
+{
+    char *path;
+
+    path = getcwd(NULL, 0);
+    if (path == NULL)
+    {
+        perror("Error getting current directory");
+        g_state = 1;
+    }
+    else
+    {
+        ft_putendl_fd(path, STDOUT_FILENO);
+        free(path);
+    }
+    return ;
+}
+
+char    *char_pwd(void) 
```

Comment:
I suggest changing name to get_cwd. Maybe reuse this function on the previous one